src/components/content-tab-previews.tsx

```diff
@@ -178,6 +178,10 @@ export function ContentTabPreviews( { selectedSite }: ContentTabPreviewsProps )
 	);
 	const isOffline = useOffline();
 
+	useEffect( () => {
+		setIsSnapshotProcessing( isUploading || isSnapshotLoading );
+	}, [ isUploading, isSnapshotLoading, setIsSnapshotProcessing ] );
```

Comment:
Let's remove this useEffect in favor of changing the state more imperatively in useArchiveSite

Reply:
Thank you, Antonio! I have made the change in https://github.com/Automattic/studio/pull/960/commits/1201b045efa28e61d201442d1519fcb4d1a0901b.